Rocket.Chat/Helpers/RCDateFormatter.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  RCDateFormatter.swift
+//  Rocket.Chat
+//
+//  Created by Luca Justin Zimmermann on 07.02.18.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import Foundation
+
+class RCDateFormatter {
+    private static let dateFormatter = DateFormatter()
+
+    static func date(_ date: Date, dateStyle: DateFormatter.Style = .medium) -> String {
+        dateFormatter.dateStyle = dateStyle
+        dateFormatter.timeStyle = .none
+        return self.dateFormatter.string(from: date)
+    }
+
+    static func time(_ date: Date, timeStyle: DateFormatter.Style = .short) -> String {
+        dateFormatter.dateStyle = .none
+        dateFormatter.timeStyle = timeStyle
+        return self.dateFormatter.string(from: date)
+    }
+
+    static func datetime(_ date: Date, dateStyle: DateFormatter.Style = .medium, timeStyle: DateFormatter.Style = .short) -> String {
+        dateFormatter.dateStyle = dateStyle
+        dateFormatter.timeStyle = timeStyle
+        return self.dateFormatter.string(from: date)
+    }
```

Comment:
Great job @soundstorm! What about having tests on this class? 🙈

Reply:
@soundstorm I merged it already, to send another TestFlight, but can you please add tests in another PR?